bitbar/ambient.60s.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+SOURCE="${BASH_SOURCE[0]}"
+while [ -h "$SOURCE" ]; do # resolve $SOURCE until the file is no longer a symlink
+  DIR="$( cd -P "$( dirname "$SOURCE" )" >/dev/null 2>&1 && pwd )"
+  SOURCE="$(readlink "$SOURCE")"
+  [[ $SOURCE != /* ]] && SOURCE="$DIR/$SOURCE" # if $SOURCE was a relative symlink, we need to resolve it relative to the path where the symlink file was located
+done
+DIR="$( cd -P "$( dirname "$SOURCE" )" >/dev/null 2>&1 && pwd )"
+
+
+WIDGET=$(/usr/local/bin/fish $DIR/../ambient-widgets | tr '\n' ' ')
+
+if [ -z "$WIDGET" ]; then
+	echo "🚄"
```

Comment:
I think it would make more sense to just not show the widget if we're not on a train.

Reply:
I agree, but I couldn't find a way to hide the widget if it is the only one in Bitbar. But that may be a problem just for me who just uses Bitbar for this.

Reply:
No, you're right, that's a known issue (see matryer/bitbar#401). Perhaps the solution then could be some sort of configuration.

Reply:
I'd gladly approve a PR adding this, once that gets resolved. :+1:

Reply:
I actually have no idea how to add preferences for this since this is mostly a question of personal style. Would that maybe a .ambient in home?

Reply:
I'd prefer using the XDG base directories, to avoid having yet another entry in home. But I agree that this can be added in a separate PR.